BundleClient/build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        agp_version = '8.5.0' // works for Android Studio Koala
+        agp_version = '8.2.2' // works for Android Studio Koala
```

Comment:
was this required?
i think @breed  just raised a PR today for changing agp to 8.5 for root project gradle

Reply:
removed this change